lib/ruby_lsp/tapioca/run_gem_rbi_check.rb

```diff
@@ -82,35 +88,46 @@ def execute_tapioca_gem_command(gems)
 
       sig { params(gems: T::Array[String]).void }
       def remove_rbis(gems)
-        FileUtils.rm_f(Dir.glob("sorbet/rbi/gems/{#{gems.join(",")}}@*.rbi"))
+        FileUtils.rm_f(Dir.glob("sorbet/rbi/gems/{#{gems.join(",")}}@*.rbi", base: project_path))
         log_message("Removed RBIs for: #{gems.join(", ")}")
       end
 
       sig { void }
       def cleanup_orphaned_rbis
-        untracked_files = %x(git ls-files --others --exclude-standard sorbet/rbi/gems/).lines.map(&:strip)
-        deleted_files = %x(git ls-files --deleted sorbet/rbi/gems/).lines.map(&:strip)
+        untracked_files = execute_in_project_path(
+          "git",
+          "ls-files",
+          "--others",
+          "--exclude-standard",
+          "sorbet/rbi/gems/",
+        ).lines.map(&:strip)
+        deleted_files = execute_in_project_path("git", "ls-files", "--deleted", "sorbet/rbi/gems/").lines.map(&:strip)
 
         delete_files(untracked_files, "Deleted untracked RBIs")
         restore_files(deleted_files, "Restored deleted RBIs")
       end
 
       sig { params(files: T::Array[String], message: String).void }
       def delete_files(files, message)
-        files.each { |file| File.delete(file) }
+        FileUtils.rm(files.map { |file| File.join(project_path, file) })
         log_message("#{message}: #{files.join(", ")}") unless files.empty?
       end
 
       sig { params(files: T::Array[String], message: String).void }
       def restore_files(files, message)
-        files.each { |file| %x(git checkout -- #{file}) }
+        files.each { |file| execute_in_project_path("git", "checkout", "--", file) }
         log_message("#{message}: #{files.join(", ")}") unless files.empty?
       end
 
       sig { params(message: String).void }
       def log_message(message)
         @stdout += "#{message}\n"
       end
+
+      def execute_in_project_path(*parts)
+        stdout, _status = T.unsafe(Open3).capture2e(*parts, chdir: project_path)
+        stdout
```

Comment:
Note that this is both stdout **and** stderr with `capture2e`. Not sure if you want both, but it might be good to change the variable name, regardless.